### ISSUE TITLE
Add "Founded by" text to the "What is The Debt Collective?" section in the landing page

### ISSUE THREAD

**Issue by orlando** [debtcollective/parent#219]
In the [current about us page](https://debtcollective.org/about) we have listed the names of the founders of the debt collective. We want to preserve this info in the new landing page.

The idea is to add this information at the bottom of the Read More text

![image](https://user-images.githubusercontent.com/849872/46250996-e37be780-c40c-11e8-80a3-76ca6bed0d0a.png)

The text we need to add is the following one:

```
The Debt Collective was founded by:

Hannah Appel Thomas Gokey Laura Hanna
Luke Herrine Ann Larson Astra Taylor
```

Preserving the order like this image 

![image](https://user-images.githubusercontent.com/849872/46251009-35bd0880-c40d-11e8-8c22-8b6ea5f9a659.png)

## Tasks

- [ ] Add founders text to the **What is The Debt Collective?** section in the landing page

## Acceptance criteria

- Founders should be listed in the **What is The Debt Collective?**
- Founders text should like the template provided above

**Comment by shoshber** [comment]
In the footer, there is an "About Us" link. Should it be changed to point to the landing page?

**Comment by orlando** [comment]
@shoshber yes, point it to the section using a hash param, like <landingurl>/#about

**Comment by shoshber** [comment]
sounds good
